spec/forms/idv/finance_form_spec.rb

```diff
@@ -5,9 +5,9 @@
 
   describe 'presence validations' do
     it 'is invalid when required attributes are not present' do
-      valid_params = { finance_type: :ccn, finance_account: '12345678' }
+      valid_params = { finance_type: :mortgage, mortgage: 'abc123' }
```

Comment:
should we also have a "happy path" test for mortgage? Right now it looks like we are only testing mortgage for the invalid state.